llvm/test/Transforms/LoopVectorize/PowerPC/large-loop-rdx.ll

```diff
@@ -10,72 +10,29 @@ define void @QLA_F3_r_veq_norm2_V(ptr noalias %r, ptr noalias %a, i32 %n) {
 ; CHECK-SAME: ptr noalias [[R:%.*]], ptr noalias [[A:%.*]], i32 [[N:%.*]]) {
 ; CHECK-NEXT:  [[ENTRY:.*]]:
 ; CHECK-NEXT:    [[CMP24:%.*]] = icmp sgt i32 [[N]], 0
-; CHECK-NEXT:    br i1 [[CMP24]], label %[[ITER_CHECK:.*]], label %[[FOR_END13:.*]]
-; CHECK:       [[ITER_CHECK]]:
+; CHECK-NEXT:    br i1 [[CMP24]], label %[[FOR_COND1_PREHEADER_PREHEADER:.*]], label %[[FOR_END13:.*]]
```

Comment:
From the comment above the function it looks like they are expecting it be interleaved 8 times, perhaps for optimal performance?

Reply:
This actually comes from fhahn's change here https://github.com/llvm/llvm-project/pull/126437/files#diff-67309aee6068e2dd0d01af33e35d15855352bb11009158c554228043cfeaa4ba